pcsx2/gui/MainFrame.cpp

```diff
@@ -637,35 +639,51 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
+	const CDVD_SourceType Source = g_Conf->CdvdSource;
+	const wxChar* SourceLabel[static_cast<uint8_t>(CDVD_SourceType::NumberOfSources)];
+	const wxChar* FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::NumberOfSources)];
+	const wxChar* BootIndicator[2][2];
+
+	FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::Iso)] = _("Boot the VM using the current ISO source media");
+	FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::Plugin)] = _("Boot the VM using the current CD/DVD source media");
+	FullBootHelpText[static_cast<uint8_t>(CDVD_SourceType::NoDisc)] = _("Boot the VM without any source media");
+
+	SourceLabel[static_cast<uint8_t>(CDVD_SourceType::Iso)] = _("ISO");
+	SourceLabel[static_cast<uint8_t>(CDVD_SourceType::Plugin)] = _("CDVD");
+	SourceLabel[static_cast<uint8_t>(CDVD_SourceType::NoDisc)] = _("BIOS");
+
+	BootIndicator[0][0] = _("Boot");
+	BootIndicator[0][1] = _("Reboot");
+	BootIndicator[1][0] = _(" (f&ull)");
+	BootIndicator[1][1] = _(" (&fast)");
+
+	wxString MenuItemLabel = static_cast<wxString>(BootIndicator[0][ActiveVM]) + " " + static_cast<wxString>(SourceLabel[static_cast<uint8_t>(Source)]);
```

Comment:
> I see, didn't know you were from China. I thought you were from UK based on your wordings on replies but guess I was wrong :P

Last I checked I was from the UK. XD

My second language does happen to be Chinese though (Cantonese/Traditional Chinese).